src/utils/values.h

```diff
@@ -117,6 +117,15 @@ int sr_val_build_str_data(sr_val_t *value, sr_val_type_t type, const char *forma
  */
 int sr_dup_val(const sr_val_t *value, sr_val_t **value_dup);
 
+/**
+ * @brief Compare two values.
+ *
+ * @param [in] value1 Sysrepo value to compare
+ * @param [in] value2 Sysrepo value to compare
+ * @param [out] 1 is the values are equal, 0 othewise
+ */
+int sr_cmp_val(const sr_val_t *value1, const sr_val_t *value2);
```

Comment:
`cmp` would suggest that the function returns `0` on equality (`strcmp`) but it does not. So please rename to `sr_equal_val` or invert the return value.